src/util.rs

```diff
@@ -105,6 +105,15 @@ pub fn ray_box_intersect(ro: Vec3<f64>, rd: Vec3<f64>, bb: Aabb<f64>) -> Option<
     }
 }
 
+/// Takes a float and wraps it around -180..180 in a way that matches the
+/// client's interpretation of yaw values
+pub fn wrap_yaw(yaw: f32) -> f32 {
```

Comment:
Do we need to make this public?

Reply:
I'm not sure. 

The `Client` and `Entity` structs are public, so the `yaw` field they possess is also public.
If someone wanted to set the yaw, they would need the function.

Perhaps there's a better way of hiding `wrap_yaw` and apply it discreetly?